src/buildHTML.js

```diff
@@ -432,12 +432,16 @@ groupTypes.genfrac = function(group, options) {
     const denomm = buildGroup(group.value.denom, newOptions, options);
 
     let rule;
+    let ruleSpacing;
+    let ruleWidth;
     if (group.value.hasBarLine) {
         rule = makeLineSpan("frac-line", options);
+        ruleWidth = ruleSpacing = rule.height;
```

Comment:
Can you change this to:
```
ruleWidth = rule.height;
ruleSpacing = ruleWidth;
```
I misread this and thought that `ruleSpacing` wasn't being defined in this block.